public/images/svg/customer-logos-zendesk.svg

```diff
@@ -1,54 +1,22 @@
-		         c-1.2,0.1-2.2,1.2-2.2,2.5c0,0.6-0.1,1.3-0.1,1.9c0,1,0,2.1,0,3.1c0,0.9-0.4,1.4-1.1,1.4c-0.7,0-1.1-0.5-1.1-1.4
-		         C47.3,22.1,47.3,21.3,47.3,20.5C47.3,20.5,47.3,20.5,47.3,20.5z"/>
-    <path class="customer-zendesk" d="M30,15.5c-0.3,0-0.5,0-0.7,0c-1.4,0-2.8,0-4.2,0c-0.6,0-1.1-0.3-1.1-0.8c0-0.5,0.4-0.9,1.1-0.9
-		         c2.3,0,4.5,0,6.8,0c0.5,0,0.9,0.1,1.1,0.6c0.2,0.5,0,0.9-0.3,1.2c-1.7,2-3.4,4-5,6c-0.2,0.2-0.3,0.4-0.5,0.7c0.3,0,0.5,0,0.7,0
-		         c1.4,0,2.8,0,4.2,0c0.7,0,1.1,0.3,1.1,0.9c0,0.5-0.4,0.9-1.1,0.9c-2.3,0-4.5,0-6.8,0c-0.5,0-0.9-0.2-1.1-0.6
-		         c-0.2-0.4,0-0.8,0.3-1.1c1.7-2,3.4-4,5-6C29.6,16.1,29.7,15.9,30,15.5z"/>
-    <path class="customer-zendesk" d="M89.3,13.4c1.1-0.1,2.2,0.2,3.3,0.7c0.6,0.3,0.9,0.7,0.6,1.2c-0.2,0.5-0.7,0.7-1.3,0.4
-		         c-1.1-0.5-2.2-0.7-3.4-0.5c-0.3,0-0.6,0.1-0.8,0.3c-0.6,0.4-0.7,1.2-0.1,1.6c0.4,0.2,0.8,0.4,1.2,0.5c0.9,0.4,1.9,0.6,2.7,1.1
-		         c1.1,0.5,1.8,1.3,1.7,2.6c-0.1,1.2-0.8,2-1.9,2.5c-2.2,0.9-4.3,0.6-6.4-0.6c-0.4-0.2-0.6-0.5-0.5-1c0.2-0.6,0.7-0.8,1.2-0.5
-		         c1,0.6,2.1,0.9,3.3,0.8c0.5,0,1-0.2,1.5-0.4c0.9-0.4,0.9-1.4,0-1.8c-0.6-0.3-1.3-0.5-2-0.8c-0.8-0.4-1.7-0.7-2.5-1.2
-		         c-1.7-1.1-1.5-3.3,0.2-4.3C87.3,13.5,88.3,13.4,89.3,13.4z"/>
-    <path class="customer-zendesk" d="M12,7.8C11.4,7,10.6,6.5,9.6,6.4c-0.1,0-0.3-0.2-0.4-0.3c-1.2-1-2.5-1.5-4-1.6C4.8,4.5,4.7,4.3,4.8,4
-		         c0.1-1.3,0.6-2.5,1.4-3.6c0.1-0.1,0.3-0.2,0.4-0.2c3.2-0.1,6.2,2.7,6.4,5.9c0,0.4,0,0.8-0.4,1C12.4,7.3,12.2,7.5,12,7.8z"/>
-    <path class="customer-zendesk" d="M15,20.1c-0.7,1.4-1.7,2.4-3.1,3.1c-0.1,0.1-0.3,0-0.5,0c-3.1-1.3-4.6-5.6-3.1-8.5c0,0,0-0.1,0.1-0.1
-		         c0.6,0.5,1.2,1,1.9,1.3c0.8,0.4,1.4,1,1.8,1.8c0.5,1,1.4,1.7,2.4,2.2C14.7,19.9,14.8,19.9,15,20.1z"/>
-    <path class="customer-zendesk" d="M12.7,16.5c0.9-0.6,1.7-1.2,2.5-1.8c0.1-0.1,0.3-0.1,0.5-0.1c1.7,0.5,3.2,0.3,4.8-0.5c0.1,0,0.1,0,0.2-0.1
-		         c0.4,0.7,0.6,1.5,0.7,2.3c0.1,0.5-0.1,1.1-0.1,1.6c0,0.5-0.3,0.7-0.7,0.9c-2.8,1.2-6.1,0.3-7.8-2.1C12.8,16.6,12.8,16.6,12.7,16.5z
-		         "/>
-    <path class="customer-zendesk" d="M6,11c0.6,0.9,1.1,1.7,1.6,2.5c0.1,0.1,0.1,0.4,0,0.5c-0.7,1.2-1.1,2.6-1,4c0,0.4,0.1,0.8,0.2,1.3
-		         c-0.2,0-0.3,0.1-0.5,0.1C5.1,19.5,4,19.4,2.9,19c-0.5-0.2-0.7-0.4-0.8-1c-0.5-2.9,1.1-5.9,3.8-7C5.9,11,6,11,6,11z"/>
-    <path class="customer-zendesk" d="M16.4,13.5c0.2-0.3,0.5-0.6,0.7-1c0.6-1,1-2,0.8-3.2c0-0.2,0.1-0.4,0.2-0.5c0.6-0.8,1-1.7,1.2-2.6
-		         c0.1-0.3,0.2-0.4,0.5-0.3c1.3,0.4,2.3,1,3.2,2c0.2,0.2,0.2,0.6,0.2,0.8c-0.6,2.6-2.9,4.6-5.5,4.9c-0.4,0-0.9,0.1-1.3,0.1
-		         C16.5,13.6,16.5,13.6,16.4,13.5z"/>
-    <path class="customer-zendesk" d="M2.2,12.3C1,11.2,0.3,10,0,8.6C0,8.4,0.1,8.2,0.2,8c1.7-2.3,5.4-3,8-1.5C8,6.6,7.9,6.6,7.8,6.7
-		         C6.7,7.2,6.2,8.1,6,9.2C6,9.7,5.8,9.9,5.3,10.1c-1.1,0.3-1.9,0.9-2.7,1.7C2.5,11.9,2.4,12.1,2.2,12.3z"/>
-  </g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This svg doesn't appear to be in use yet, and this alteration helps it line up better with the other customer logo svgs